app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,11 @@
+from app.players.player import Player
+from abc import ABC
+
+
+class Dwarf(Player, ABC):
```

Comment:
Do not repeat abstract methods from a parent if you have an abstract class that inherits another abstract class.
Since `Player` is already a child of the `ABC` class, you can inherit just from `Player`.